cpp/include/cudf/utilities/traits.hpp

```diff
@@ -331,6 +331,18 @@ constexpr inline bool is_timestamp(data_type type)
   return cudf::type_dispatcher(type, is_timestamp_impl{});
 }
 
+/**
+ * @brief Function for determining if `type_id` is fixed_point
+ *
+ * @param id The `type_id` to check
+ * @return true If id is either DECIMAL32 or DECIMAL64
+ * @return false If id is not either DECIMAL32 or DECIMAL64
+ */
+constexpr inline bool is_fixed_point(type_id id)
+{
+  return id == type_id::DECIMAL32 || id == type_id::DECIMAL64;
```

Comment:
In order to have a single source of truth, I think this function should follow the same pattern of other traits that operate on `type_id`s and invoke the `type_dispatcher` to use the `is_fixed_point<T>` trait defined below.

Reply:
Done. Was able to remove it and just use the existing `is_fixed_point`